config/prod/cis-for-beanstalk-image-pipeline.yaml

```diff
@@ -0,0 +1,9 @@
+template:
+  type: "http"
+  url: "https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.9.6/templates/ImageBuilder/cis-for-eb-image-pipeline-template.yaml"
```

Comment:
This base template has a pipeline that generates AMIs weekly.  My concern is that the storage cost may become expensive over time.  Have you looked into the implications of cost over time for that pipeline?

Reply:
you'll probably need to bump the version here to pick up PR https://github.com/Sage-Bionetworks/aws-infra/pull/428

Reply:
> you'll probably need to bump the version here to pick up PR [Sage-Bionetworks/aws-infra#428](https://github.com/Sage-Bionetworks/aws-infra/pull/428)

Yes, I updated the PR from referencing `v0.9.6` to `v0.9.8` , but thanks for the reminder.